### ISSUE TITLE
Community page Accessibility: [MAS1.4.3]:Text elements must have sufficient color contrast against the background.

### ISSUE THREAD

**Issue by tylerbutler** [microsoft/FluidFramework#3816]
**Build details:**

**OS Version:**2004(20221.1000)

**Edge Dev Version**: 87.0.654.0 (Official build) dev (64-bit)

**URL:** <https://fluidframework.com/community/>

**Pre-Requisite:**

Add Accessibility Insights extension for web to Edge Anaheim (url:[https://accessibilityinsights.io/](https://accessibilityinsights.io/))

  

**Repro Steps:**

1\. Open the given URL with valid credentials.

2.Now, Run fastpass using Accessibility Insights for Web and observe the failure instances of color-contrast

  
**Actual Result**  
Text elements are not having sufficient color contrast ratio.  

  
**Expected Result:**  
Text elements must have sufficient color contrast against the background.  

  

**User Impact:**  

Some people with low vision experience low contrast, meaning that there aren't very many bright or dark areas. Everything tends to appear about the same brightness, which makes it hard to distinguish outlines, borders, edges, and details. Text that is too close in luminance (brightness) to the background can be hard to read.  

  

**MAS Reference:**

**MAS 1.4.3 - Contrast (Minimum)**

**Comment by micahgodbolt** [comment]
Specifically these links

![image](https://user-images.githubusercontent.com/1434956/95385543-fef31600-08a2-11eb-8203-f4361316f34f.png)

We need to either find an alternative link style for text on this dark bg, or find an alternative bg for this block of text.
